SUMMARY.md

```diff
@@ -39,8 +39,9 @@
     * [Choose attach document API](buyer/issue/attach-document-api.md)
     * [No delivery expected](buyer/issue/no-delivery-expected.md)
   * [Update an existing order](buyer/update.md)
-  * [Receive an order response](buyer/webhook/README.md)
-    * [Download a document attached to an order response](buyer/webhook/download-document.md)
+  * [Receive an order response](buyer/receive/README.md)
+    * [Choose webhook or polling API](buyer/receive/receive-order-response-api.md)
```

Comment:
Why do we have both of the `Choose webhook or polling API` pages? Is it possible to extract comon part in the single page ?

Reply:
@olegtradecloud Good idea, I moved it to api/webhook-vs-polling.md

Reply:
Thanks